lib/suspenders/generators/advisories_generator.rb

```diff
@@ -3,7 +3,11 @@
 module Suspenders
   class AdvisoriesGenerator < Generators::Base
     def bundler_audit_gem
-      gem "bundler-audit", require: false, group: %i[development test]
+      gem "bundler-audit",
+          require: false,
+          group: %i[development test],
+          git: "https://github.com/rubysec/bundler-audit.git",
+          branch: "0.7.0"
```

Comment:
Pinning bundler-audit to 0.7.0 until it's released. It has the bugfix that allows it to work with the latest version of Rails.

If you hate this feel free to revert this commit.